README.md

```diff
@@ -1,91 +1,156 @@
-# NHSE/I: Wagtail Test
+# NHSEI Website
 
-This is used to test our IMPORTER app inside a wagtail instance and is were we are collecting the data from SCRAPY and importing it into a Wagtail instance.
+This is the repo for the NHS England & Imporvement website https://england.nhs.uk
 
-We expect this app to show us the result of imporing pages, categories, posts and blog  and give us a chance to test out theories etc. It's an ongoing project that will likely chnage many times before production use as we discover data that needs conversion to a more suitable format to Wagtail import.
+Currently available at https://nhsei-staging.uksouth.cloudapp.azure.com
 
-## Running the scripts
+# Developer Install Guide
 
-# Importing
+Notes: 
 
-The data is collected from https://nhsei-scrapy.rkh.co.uk/api/ 
+* In production the site runs in a docker container but in developent it can be more conveininet to run the site in your local environment.
```

Comment:
```suggestion
* In production the site runs in a docker container but in development it can be more convenient to run the site in your local environment.
```